### PR TITLE
fix(*): add mainnet network checks

### PR DESCRIPTION
**Pull request by sideninja**
Update some missing mainnet values.

issue: none

### CODE REVIEW

File: monitor/routerecon/recon.go [hunk 1]
```diff
@@ -19,7 +19,7 @@ import (
 )
 
 func ReconForever(ctx context.Context, network netconf.Network, xprov xchain.Provider, ethCls map[uint64]ethclient.Client) {
-	if network.ID != netconf.Omega {
+	if !network.ID.IsProtected() {
```
Comment (kevinhalliday) [inline monitor/routerecon/recon.go:22]: is mainnet routescane setup yet? we'll need to change the urls we use

Comment (sideninja) [reply monitor/routerecon/recon.go:22]: yeah that's the part I don't have context for, so wanted to check

**Review by corverroos — APPROVED** [review]

File: monitor/routerecon/recon.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (sideninja) [reply monitor/routerecon/recon.go:22]: is it then better to rollback this part or is it fine? sorry not sure exactly if it's not used.

[the remark this answers is not in the document]
Comment (corverroos) [reply monitor/routerecon/recon.go:22]: remove this chage

**Pull request merged**